src/server/config/kibana.yml

```diff
@@ -61,7 +61,7 @@ verify_ssl: true
 
 # If you would like to send the log output to a file you can set the path below.
 # This will also turn off the STDOUT log output.
-# log_file: ./kibana.log
+log_file: /var/log/probe/kibana.log
```

Comment:
Good call @john-gress regarding the permissions. 

@craig-cogdil, @john-gress Kibana doesn't seem to support natively log rotation. It can be achieved with a 3rd party log tool. Ref: https://discuss.elastic.co/t/kibana-logs-rotation/66441

So then the question is how much the log is spitting out? I haven't tested it myself yet but without log rotation it could become an issue. 

I'm suggesting (part of it what John mentioned).
1. Create a story to address this. 
2.   -- log rotation is a must have (ref the link I posted)
3.  UI access to see the log.